src/tests/alerts/specs/search_alerts_tests.jsonl

```diff
@@ -1,26 +1,26 @@
-{"id":"ui_PZbaYHYFD_fgoVjkzd","clusterStateId":"alerting","question":"How many alerts were triggered on Dec 05 2023?","expectedAnswer":"1 alert was triggered on Dec 05 2023. The alert has ID uzg0gosB6jqYe1T0y9Aa and was triggered by monitor random_query_monitor."}
-{"id":"ejbPZp9WEgYiyUH7VwPwk","clusterStateId":"alerting","question":"What are the names of the monitors that triggered alerts on Dec 10 2023?","expectedAnswer":"2 monitors triggered alerts on Dec 10 2023 - random_doc_monitor and random_comp_monitor."}
-{"id":"B_aIDTWPmk0VnO1bfFfNk","clusterStateId":"alerting","question":"Have any alerts been acknowledged in the last 3 days?","expectedAnswer":"Yes, 1 alert has been acknowledged in the last 3 days. The alert with ID GThUkYsB6jqYe1T0UOF3 was acknowledged on Dec 08 2023 09:00."}
+{"id":"ui_PZbaYHYFD_fgoVjkzd","clusterStateId":"alerting","question":"How many alerts were triggered on {{six days, four hours, 50 minutes, and 21 seconds ago}}?","expectedAnswer":"1 alert was triggered on {{six days, four hours, 50 minutes, and 21 seconds ago}}. The alert has ID uzg0gosB6jqYe1T0y9Aa and was triggered by monitor random_query_monitor."}
```

Comment:
would adding minutes and seconds make it flaky?

Reply:
Wrote up a script to test it. It looks like `chrono-node` is a bit flaky when the string is not grammatically correct. For example, it's not very consistent when the syntax is `"x days and x hours and x minutes and x seconds"`. When adjusted to be grammatically correct with the Oxford comma (`"x days, x hours, x minutes, and x seconds"`), it's consistent. Results are in a gist [here](https://gist.github.com/sejli/bb9281d9ce9f12062a46aa39f200ba00).

Reply:
that's interesting. maybe add the grammar comma requirement to developer guide as well to avoid surprises?

Reply:
Updated documentation.